v2/sunshine/notely_app/assets/user_sidebar.html

```diff
@@ -49,23 +49,37 @@
           function recordActivity() {
             var note = document.getElementById('note');
             var type = document.getElementById('type');
-            var description = '[' + type.value + '] ' + note.value;
+            var description = '[' + type.text + '] ' + note.value;
 
             if (note.value === '') return;
 
             var settings = {
-              url: '/api/cdp/track',
+              url: '/api/v2/users/'+ profileId + '/events',
```

Comment:
To be more explicit, suggest changing this to be make it obvious that this is a userId and not actually a profileId:
```suggestion
              url: '/api/v2/users/'+ userId + '/events',
```

Reply:
Done